### PR TITLE
Fix 'Share to Reddit' link for Google brews

### PR DESCRIPTION
**Pull request by G-Ambatte**
First pass at fixing the issue where 'Share to Reddit' is generating incorrect links for brews stored on Google Drive.

### CODE REVIEW

**Comment by G-Ambatte** [comment]
Initial testing appears to be successful. Promoting to Ready for Review.

File: client/homebrew/editor/metadataEditor/metadataEditor.jsx [hunk 1]
```diff
@@ -67,10 +67,12 @@ const MetadataEditor = createClass({
 
 	getRedditLink : function(){
 		const meta = this.props.metadata;
+
+		const shareLink = (meta.googleID ? meta.googleId + meta.shareId : meta.shareId);
```
Comment (calculuschild) [inline client/homebrew/editor/metadataEditor/metadataEditor.jsx:71]: Just a slight change.

```suggestion
		const shareLink = meta.googleID ? meta.googleId + meta.shareId : meta.shareId;
```

Or.... if you think this is cleaner:

```suggestion
		const shareLink = (meta.googleID || '') + meta.shareId;
```

**Review by calculuschild — CHANGES_REQUESTED** [review]

File: client/homebrew/editor/metadataEditor/metadataEditor.jsx (hunk 1 above)
Comment (G-Ambatte) [reply client/homebrew/editor/metadataEditor/metadataEditor.jsx:71]: I think both have their merits, but the second option is cleaner.

File: client/homebrew/editor/metadataEditor/metadataEditor.jsx [hunk 2]
```diff
@@ -67,10 +67,12 @@ const MetadataEditor = createClass({
 
 	getRedditLink : function(){
 		const meta = this.props.metadata;
+
+		const shareLink = (meta.googleID || '') + meta.shareId;
```
[the remark this answers is not in the document]
Comment (calculuschild) [reply client/homebrew/editor/metadataEditor/metadataEditor.jsx:71]: Good grief. Apparently I never clicked the "submit review" button and this has just been sitting here for weeks.

**Review by calculuschild — APPROVED** [review]

**Pull request merged**